### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by wlandau**
This PR updates the documentation on https://r-multiverse.org, partly to bring it up to date, and partly to organize and systematize the information. In addition, I made changes to avoid the appearance of criticizing traditional repositories, and I mentioned the new [status](https://r-multiverse.org/status/) and [topics](https://r-multiverse.org/topics/) systems. It would be great to review this in our admin meeting tomorrow.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -1,13 +1,37 @@
 ---
-title: "Community"
+title: "Community repository"
 ---
 
-## An Open Repository for R Packages
+The R-multiverse Community repository democratizes the distribution and deployment of R package releases.
```
Comment (maelle) [inline community.md:5]: is "democratize" too vague? "simplifies"?

File: community.md [hunk 2]
```diff
@@ -16,19 +40,9 @@ R-multiverse always provides a version that its developer chose to distribute fo
...
-## What about my personal R-universe?
-
-R-multiverse perfectly complements your existing personal or organization R-universe.
+## Personal universes
 
-This can continue to hold either development or release versions for yourself and your colleagues.
-At the same time, R-multiverse deploys the latest package releases directly to the wider community.
+You can deploy a package through a personal or [organization](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations)-level
+universe even if the package also is part of R-multiverse.
+In fact, a personal universe may help reproduce and diagnose `R CMD check` issues
```
Comment (maelle) [inline community.md:47]: later maybe show an actual example of a package with such a setup / or add a diagram?

File: contributors.md [hunk 3]
```diff
@@ -2,90 +2,107 @@
...
+To register a package with R-multiverse:
 
 1. Navigate to <https://github.com/r-multiverse/contributions>.
-2. Open a [GitHub pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) to add one or more listings to the [`packages` folder](https://github.com/r-multiverse/contributions/tree/main/packages).
+1. Contribute a [package listing](https://github.com/r-multiverse/contributions/tree/main/packages) via
+[GitHub pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
 
 Watch this 1-minute video to learn how:
 
 {{< video https://vimeo.com/923333903 title="R-multiverse contribution tutorial" >}}
```
Comment (maelle) [inline contributors.md:30]: could this video get subtitles?

File: contributors.md [hunk 4]
```diff
@@ -2,90 +2,107 @@
...
+in the title of the pull request.
 
-```md
-[![R-multiverse status](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fcommunity.r-multiverse.org%2Fapi%2Fpackages%2FpkgNAME&query=%24.Version&label=r-multiverse)](https://community.r-multiverse.org/pkgNAME)
-```
+## Community
+
+After registration, the latest release of the package will
+automatically appear in the [Community](community.md) repository.
+[GitHub Actions](https://github.com/features/actions) workflows periodically
```
Comment (maelle) [inline contributors.md:54]: In this paragraph it seems the line breaks are not semantic. (I'm still working on https://github.com/maelle/aeolus, in a few weeks it should be usable)

File: contributors.md [hunk 5]
```diff
@@ -2,90 +2,107 @@
...
-<https://community.r-multiverse.org> periodically refreshes to include the latest release of each package in `packages.json`.
+## Deregistration
 
-## Ownership and attribution
+If it becomes necessary to deregister a package from R-multiverse, you may submit a
+[pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+to remove the [package listing](https://github.com/r-multiverse/contributions/tree/main/packages).
+To prevent a package of the same name from registering in R-multiverse,
+you may contribute a listing with free-form text (not a URL or JSON list).
+Packages with missing or free-form listings are removed from [Community](community.md), and the subsequent [Production](production.md) snapshot will not include them.
```
Comment (maelle) [inline contributors.md:102]: even if it's sort of obvious, add that the package is never removed from past snapshots?

File: index.md [hunk 6]
```diff
@@ -2,25 +2,58 @@
...
 ---
 
-<img src="logo/logo-hexbin.png" align="right" width="140" />
+R-multiverse is a general-purpose R package repository system with
+[transparent governance](policies.md) and open source infrastructure.
+R-multiverse provides:
 
-#### A community-curated package repository system, powered by <br />R-universe
+1. A home for packages that fall outside the
+[CRAN Repository Policy](https://cran.r-project.org/web/packages/policies.html).
```
Comment (maelle) [inline index.md:10]: outside the scope of other repositories such as CRAN or Bioconductor?

File: production.md [hunk 7]
```diff
@@ -1,58 +1,126 @@
...
+The [Staging](#staging) repository is active during the month-long period prior to each snapshot.
+During that time, [Staging](#staging) stabilizes the Production candidates
+while still allowing bug fixes.
 
-## The Production Staging Process
+While [Staging](#staging) is active, if a package is failing one or more [R-multiverse checks](#checks),
+then new releases of that package are continuously pulled from [Community](community.md).
+Otherwise, [Staging](#staging) freezes the package at its current release
+and no longer accepts updates from [Community](community.md).
+This freeze prevents new problems in reverse dependencies downstream.
```
Comment (maelle) [inline production.md:50]: for such things, later, diagrams would be nice (not now)

**Review by maelle — COMMENTED** [review]
:rocket:

File: community.md (hunk 1 above)
Comment (wlandau) [reply community.md]: edited in 9fc7d0994d4aed5050b5ca18d7ea4ac4e0799f9c

File: production.md [hunk 8]
```diff
@@ -1,58 +1,126 @@
...
+    "branch": "a76b8d56e6f39a6157880069f9d32f3cc1f574d7"
+  },
+```
 
-The currently available packages may be queried using:
+then you can install the version of that dependency from R:
 
 ```r
-available.packages(repos = "https://production.r-multiverse.org")
+remotes::install_github(
```
Comment (maelle) [inline production.md:109]: use pak

File: community.md (hunk 2 above)
Comment (wlandau) [reply community.md]: 8b085bc4612020e2480092595e9b5de7633619d0 adds a link to where the Production page elaborates on this.

File: contributors.md (hunk 3 above)
Comment (wlandau) [reply contributors.md:30]: As we discussed today, it's a visual demo, no audio.

File: contributors.md (hunk 4 above)
Comment (wlandau) [reply contributors.md:54]: I'll keep this in mind for future PRs.

File: contributors.md (hunk 5 above)
Comment (wlandau) [reply contributors.md:102]: Added a note in 6f5a52b08e0c7a8bc2d3566d94afdf0c9721b77f.

File: production.md (hunk 7 above)
Comment (wlandau) [reply production.md:52]: With a good enough diagram, I agree. Maybe a bit much for this PR.

File: production.md (hunk 8 above)
Comment (wlandau) [reply production.md]: added in 97e0bd353a584eb8c8972b03add2641ddf05b236

File: index.md (hunk 6 above)
Comment (wlandau) [reply index.md]: added in da7b8642c27974b6e25d50ee547489971d0b5a73

**Comment by shikokuchuo** [comment]
Thank you @wlandau, this looks great!

**Pull request merged**